src/applications/disability-benefits/781/routes.jsx

```diff
@@ -0,0 +1,13 @@
+import { createRoutesWithSaveInProgress } from '../../../platform/forms/save-in-progress/helpers';
+import formConfig from './config/form';
+import App from './containers/App.jsx';
```

Comment:
Instances of `App` are typically qualified, such as `Form781App`.

Reply:
Thanks, makes sense.